config/v1/types_build.go

```diff
@@ -97,6 +105,14 @@ type BuildOverrides struct {
 	// tolerations set on a build pod.
 	// +optional
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
+
+	// ForcePull specifies whether to always check for a newer version of the image
+	// in the registry (true) or allow the use of the local image (if it exists)
+	// even if it is not the latest version (false).
+	// Enabling forcePull is only relevant for tag references since a SHA reference
+	// can by definition only refer to a specific version of an image.
+	// +optional
+	ForcePull *bool `json:"forcePull,omitempty"`
```

Comment:
Per @sttts and my comment above, this doesn't need to be a pointer.

Reply:
why not?  how will you distinguish between "admin set the override to false, set all builds to false" and "admin didn't set it to anything, don't touch builds"?

Reply:
@bparees I think this comment was from before scrum and before your comment above about using pointers, so i think it just needs to be marked as resolved.

Reply:
sounds good

Reply:
I miss what @bparees writes above in the godoc. Now it sounds like only true makes sense here. Better:

`forcePull overrides, if set, the equivalent value in the builds, i.e. false disables force pull for all builds, true enables force pull for all builds, independently what each build specifies itself.`

Reply:
updated in current push